app/presenters/job_presenter.rb

```diff
@@ -26,6 +26,17 @@ def list(jobs)
         single(job)
       end.compact
     end
+    
+    def list_by_cluster_id(jobs, cluster_id)
+      jobs.map do |job|
+        next if job['payload'].blank?
```

Comment:
https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals is what I'm talking about. See the section on loops.